justfile

```diff
@@ -18,10 +13,6 @@ fmt:
 lint clipy_args=_default:
    {{ _cargo }} clipy {{ clipy_args }}
 
-#run controller
-run:
-   {{ _cargo }} run
```

Comment:
Do we need to `just run` when we can just `cargo run`? This is useful if we include a bunch of config params we want to abstract. e.g. if we had examples as part of our crate. Or maybe if there's a bunch of flags we need to pass in for compilation. Let's add stuff when it's needed.